app/common/directives/filter-system/filter-searchbar.js

```diff
@@ -22,23 +22,15 @@ function ($timeout, PostFilters) {
                     $scope.searchResultsVisible = false;
                 }, 100);
             };
-
-            $scope.enableQuery = function () {
```

Comment:
This should never have been here anyway, because we use this in settings/users too